modulefiles/gsi_common.lua

```diff
@@ -4,7 +4,7 @@ Load common modules to build GSI on all machines
 
 local netcdf_ver=os.getenv("netcdf_ver") or "4.7.4"
 
-local bufr_ver=os.getenv("bufr_ver") or "11.5.0"
+local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
```

Comment:
@RussTreadon-NOAA 
You change looks good.  I have a related question.
NCO asked us to change the following 
```
craype 2.7.10 -> 2.7.13
Prgv-Intel 8.1.0 -> 8.2.0
cray-mpich 8.1.9 -> 8.1.12
cray-pals 1.0.17 -> 1.1.3
bufr 11.4.0 -> 11.7.0
```
Shall we also update the cray stack and Prgv-Intel as well?

Reply:
@emilyhcliu , you are right.   `gsi_wcoss2.lua` currently defaults to `PrgEnv_intel/8.1.0`, craype/2.7.8`, and `cray_mpich/8.1.7`.    I'll update these to the versions NCO requested.  

 `cray-pals` is not used in the gsi build.   This module is referenced at the global-workflow level.

Reply:
@RussTreadon-NOAA 
Great!   As soon as you updated the versions NCO requested.  I will merge your PR to release/gfsda.v16.3.0
We do not have GSI code manager now.  We will have to identify a person for the role.

Reply:
Whether an individual or a team of individuals, MDAB-DAQC and EIB management will need to discuss how to handle [NOAA-EMC/GSI](https://github.com/NOAA-EMC/GSI) moving forward.  Several operational systems still pull code from [NOAA-EMC/GSI](https://github.com/NOAA-EMC/GSI).   [NOAA-EMC/GSI](https://github.com/NOAA-EMC/GSI) has several open [PRs](https://github.com/NOAA-EMC/GSI/pulls) and many more open [issues](https://github.com/NOAA-EMC/GSI/issues).   It's an active repository.

Reply:
EMC and NCO decided not to update
```
craype 2.7.10 -> 2.7.13 
Prgv-Intel 8.1.0 -> 8.2.0 
cray-mpich 8.1.9 -> 8.1.12 
cray-pals 1.0.17 -> 1.1.3
```
Revert this change at 0a289712